tla-io/src/main/scala/at/forsyte/apalache/io/quint/Quint.scala

```diff
@@ -331,6 +331,18 @@ class Quint(moduleData: QuintOutput) {
         case "item"   => binaryApp(opName, tla.app)
         case "tuples" => variadicApp(tla.times)
 
+        // Maps (functions)
+        // case "Map"       => null // Rewire with "Apalache!SetAsFun"
+        // case "Map"       => variadicApp(args => tla.appOp("Apalache!SetAsFun", tla.enumSet(args: _*)))
+        case "get"       => binaryApp(opName, tla.app)
+        case "keys"      => unaryApp(opName, tla.dom)
+        case "mapBy"     => null
```

Comment:
This should be mapped to https://github.com/informalsystems/apalache/blob/a398cff9f97fb7221d973e4780d46bd85c387f7e/tlair/src/main/scala/at/forsyte/apalache/tla/lir/oper/TlaFunOper.scala#L56-L69